### PR TITLE
Fix multicluster preview deploy

### PR DESCRIPTION
**Pull request by jLopezbarb**
# Proposed changes

Fixes https://github.com/okteto/app/issues/5460

- Use the preview used by the user in the flags instead of setting it in the context and using it across the code.
- @maroshii can you create a pipeline installer with these changes. I think the only that could recreate the namespace is creating a namespace, but I'll need to check it out before making the changes

### CODE REVIEW

File: cmd/preview/deploy.go [hunk 1]
```diff
@@ -198,7 +196,7 @@ func (pw *Command) waitUntilRunning(ctx context.Context, name, namespace string,
 	return nil
 }
 func (pw *Command) waitToBeDeployed(ctx context.Context, name string, a *types.Action, timeout time.Duration) error {
-	return pw.okClient.Pipeline().WaitForActionToFinish(ctx, name, okteto.Context().Namespace, a.Name, timeout)
+	return pw.okClient.Pipeline().WaitForActionToFinish(ctx, name, name, a.Name, timeout)
```
Comment (ifbyol) [inline cmd/preview/deploy.go:199]: Is this correct? Does the name contains the namespace to be used to query the API?

Comment (jLopezbarb) [reply cmd/preview/deploy.go:199]: Yes, If we use the namespace stored in the context is not going to be the preview env anymore...

Comment (ifbyol) [reply cmd/preview/deploy.go:199]: but the flag `--wait` won't work. I mean, that namespace is being used to do a query to the API

Comment (jLopezbarb) [reply cmd/preview/deploy.go:199]: name is the preview namespace, it should work as before

**Comment by jLopezbarb** [comment]
@maroshii This PR will be merge into your PR. Let me know when this should be merged

**Review by maroshii — APPROVED** [review]

**Pull request merged**